sympy/logic/boolalg.py

```diff
@@ -1274,14 +1343,52 @@ def _distribute(info):
             return info[0]
         rest = info[2](*[a for a in info[0].args if a is not conj])
         return info[1](*list(map(_distribute,
-            [(info[2](c, rest), info[1], info[2]) for c in conj.args])))
+            [(info[2](c, rest), info[1], info[2]) for c in conj.args])),\
+                remove_true=False)
     elif isinstance(info[0], info[1]):
         return info[1](*list(map(_distribute,
-            [(x, info[1], info[2]) for x in info[0].args])))
+            [(x, info[1], info[2]) for x in info[0].args])),\
+                remove_true=False)
     else:
         return info[0]
 
 
+def to_anf(expr, deep=True):
+    """
+    Converts expr to Algebraic Normal Form (ANF).
+
+    ANF is a canonical normal form, which means that two
+    equivalent formulas will convert to the same ANF.
+
+    A logical expression is in ANF if it has the form
+    ((A & B & ...) ^ (B & C ...) ^ B ^ True ^ ...),
```

Comment:
Would be rendered good if you can use math rendering here.

Reply:
What is math rendering? Is it possible to use it with an arbitrary absolutely abstract expression which can't be presented as a finite expression?

Reply:
Done.

    .. math::
        1 \oplus a \oplus b \oplus ab \oplus abc